content/docs/getting-started/fluvio-cli.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Installing the Fluvio CLI
+menu: Fluvio CLI
+toc: true
+weight: 20
+---
+
+The Fluvio CLI (or _command-line interface_) is an all-in-one tool for setting
+up, managing, and interacting with Fluvio. You can download the CLI from our
+[github releases] page. You'll need to open the archive, then move the `fluvio`
+executable into a directory in your `PATH`.
+
+~> Please note that we currently only support Linux and MacOS
+
+[github releases]: https://github.com/infinyon/fluvio/releases
+
+```shell
```

Comment:
need to add steps for making executable:
```
chmod +x ./fluvio
```

Reply:
When I downloaded from the releases page, the executable I got already had +x